src/enums/stat-names.enum.ts

```diff
@@ -0,0 +1,14 @@
+export enum StatNamesEnum {
+  HEALTHPOINTS = 'Health Points',
+  AETHERISPOINTS = 'Ætheris Points',
```

Comment:
SNAKE_SCREAM_CASE => a "_" is missing instead of nothing to mark the space

Reply:
I will make this reptile suffer.